### PR TITLE
Adding Changes 

### PR DESCRIPTION
**Pull request by melashu**
## Newly added files 
- css/styles.css file, several modification happened in this file 
- index.html file 
- image files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
+    <main class="container">
+        <header class="main-header">
+            <p class="dev-name">Mehsu Amare</p>
```
Comment (omarsalem7) [inline index.html:16]: - [ ] The logo for your website should be a link, kindly use `a` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"
+                    fill="white" />
+            </svg>
```
Comment (omarsalem7) [inline index.html:21]: - [ ] The hamburger menu should be wrapped in a `<nav>` tag to improve SEO.

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.main-header {
+  display: flex;
+  flex-direction: row;
+  background: #3c3a39;
+  justify-content: space-around;
```
Comment (omarsalem7) [inline css/styles.css:10]: - [ ] kindly use `justify-content: space-between; padding: 0 15px;` to make your toolbar match the design.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                experience! If you like what you
+                see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="social-media">
+                <a href="#"><img src="image/git.png" alt="github"></a>
+                <a href="#"><img src="image/linkedin.png" alt="linkedin"></a>
+                <a href="#"><img src="image/take.png" alt="github"></a>
+                <a href="#"><img src="image/twitter.png" alt="twitter"></a>
+                <a href="#"><img src="image/medium.png" alt="medium"></a>
+            </div>
```
Comment (omarsalem7) [inline index.html:36]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="arts">
+                <img src="image/arts.png" alt="art image">
+            </div>
```
Comment (omarsalem7) [inline index.html:39]: - [ ] The section has the proper background image (decorations are always background images, not img tags), kindly fix it and use background-image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+            </svg>
+        </header>
+        <section class="headline">
+            <h1 class="title">Hey There. I'm Melashu</h1>
+            <p class="supporting">I’am a Software Developer</p>
+            <p class="description">I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you
+                see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="social-media">
```
Comment (omarsalem7) [inline index.html:30]: - [ ] Kindly use a correct font-family, you can get it from Figma

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @melashu 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Align content with flex box ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (danmainah) [reply index.html:21]: - [ ] @melashu great work implementing the changes requested, kindly ensure your hamburger button is wrapped in a `nav` tag as previously requested.

Comment (danmainah) [reply index.html:21]: - [ ] Kindly ensure your design matches the design given, currently, the background image is missing.Also the font in your paragraph doesn't match the design given, below is a comparison.


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @melashu , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by melashu** [comment]
Thank you for your comment.I will fix the issues.But the font family putted in figma Leads to linter error.That why I use other font family.Thanks in advance On Jul 1, 2022 10:41 AM, Daniel Maina ***@***.***> wrote:

@danmainah requested changes on this pull request.



Hi @melashu ,

Great work on making the changes requested by a previous reviewer 👏  👏 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Highlights

✔️ No linter errors

✔️ Github flow was correctly used

Required Changes ♻️

Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.



As described in the Code reviews limits policy  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.



In index.html:

> +            <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">

+                <path fill-rule="evenodd" clip-rule="evenodd"

+                    d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"

+                    fill="white" />

+            </svg>



 @melashu great work implementing the changes requested, kindly ensure your hamburger button is wrapped in a nav tag as previously requested.







In index.html:

> +            <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">

+                <path fill-rule="evenodd" clip-rule="evenodd"

+                    d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"

+                    fill="white" />

+            </svg>



 Kindly ensure your design matches the design given, currently, the background image is missing.Also the font in your paragraph doesn't match the design given, below is a comparison.

Figma DesignYour design

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @melashu    :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes that were requested by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._## Hello @DebbyMiressa   :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes that were requested by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**